implementations/rust/ockam/ockam_api/src/nodes/service/manager.rs

```diff
@@ -288,30 +288,27 @@ impl NodeManager {
         transport_options: NodeManagerTransportOptions,
         trust_options: NodeManagerTrustOptions,
     ) -> ockam_core::Result<Self> {
+        let mut cli_state = general_options.cli_state;
+        let node_name = general_options.node_name.clone();
+        cli_state.set_node_name(&node_name);
+
         debug!("create transports");
+        tokio::time::sleep(Duration::from_secs(1)).await;
```

Comment:
Why do we need a `sleep` here? Doesn't that mean that we are adding a latency of `1 second` to many commands using an `InMemoryNode`?

Reply:
Sorry, that's something I forgot to remove. I was testing spinner messages in the nodemanager init function.